### DISCUSSION TITLE
Error in getSpatialDataset

### DISCUSSION THREAD

**Discussion by Umaarasu** [giotto-suite/Giotto#661] [General]
Hi,

When I run this command 
getSpatialDataset(dataset = 'scRNA_mouse_brain', directory = results_folder). I get this error "Error in getSpatialDataset(dataset = "scRNA_mouse_brain", directory = results_folder) : 
  object 'wrap_msg' not found". Has anyone else had this problem?

**Comment by RubD** [comment]
Hi @Umaarasu,

What's the version of GiottoData that you're using and can you try re-installing GiottoData again?

`packageVersion('GiottoData')`

`devtools::install_github("drieslab/GiottoData")`

**Reply by Umaarasu** [reply]
Hi @RubD  and @mattobny ,

> packageVersion('GiottoData')
[1] ‘0.2.1’

Even if I did re-install the GiottoData package and load the library, I get error in the next couple of steps 

> rank_matrix = makeSignMatrixRank(sc_matrix = get_expression_values(giotto_SC,values = "normalized",output = "matrix"),
                            sc_cluster_ids = pDataDT(giotto_SC)$Class)
Error in get_expression_values(giotto_SC, values = "normalized", output = "matrix") : 
  could not find function "get_expression_values"..

Similar issue with Page enrichment..the scran package is not installed along wth giotto and when I do install seperately, it gives non-zero exit status.

**Comment by mattobny** [comment]
@Umaarasu Hello, `wrap_msg()` is a function in the `Giotto` package, i.e. this repository. Loading `GiottoData` requires loading of `Giotto`. This should happen automatically, but if you are calling the function without loading the libraries, e.g. `GiottoData::getSpatialDataset()`, this could be causing the issue.

**Comment by mattobny** [comment]
@Umaarasu We're going to move this to the issues page, as this seems less like a discussion and more of an issue for troubleshooting. I'll tag you and this discussion in the new issue for reference.
